public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/SmartAlertTypeDetector.tsx

```diff
@@ -47,6 +47,11 @@ const getCanSwitch = ({
   ruleFormType: RuleFormType | undefined;
   editingExistingRule: boolean;
 }) => {
+  // we don't say anything about switching when editing existing rules
+  if (editingExistingRule) {
+    return false;
```

Comment:
maybe we could show the alert type somehow? I'm thinking about if knowing the rule type could be useful when editing 🤔